src/Modal/Header/Header.component.tsx

```diff
@@ -15,16 +25,20 @@ export const StyledHeader = styled.div<ModalHeaderProps>`
     min-height: 4rem;
     padding: 1.25rem 4.75rem 1.25rem 2rem;
     width: 100%;
-    border-radius: ${sizes.border.radius.modal} ${
-    sizes.border.radius.modal
-} 0 0;
     margin-bottom: 1rem;
 
     display: flex;
     order: -1;
 
-    color: ${({ color = colors.charcoal.light }) => color};
-    background: ${({ background = colors.white.base }) => background};
+    ${compose(
+        colorStyles,
+        backgroundStyles,
+        borderRadiusStyles
+    )}
+
+    && {
+        ${paddingStyles}
```

Comment:
What's this doing?

Reply:
I separated out the padding like this because the padding is normally defined by the parent component (Modal). This  increases the specificity so that this css takes precedence if it exists.

Reply:
Got it. Neat.